zebra/rt_netlink.c

```diff
@@ -1663,6 +1666,332 @@ static int netlink_route_multipath(int cmd, const struct prefix *p,
 			    0);
 }
 
+/*
+ * Routing table change via netlink interface, using a dataplane context object
+ */
+static int netlink_route_multipath_ctx(int cmd, dplane_ctx_h ctx)
```

Comment:
I'm not a fan of this, we have a function that does this work already.  Modify that one please instead of copy/pasting a complete copy.